cmds/console.c

```diff
@@ -26,32 +26,31 @@ static void cmd_consoleInfo(void)
 }
 
 
-static int cmd_console(char *s)
+static int cmd_console(int argc, char *argv[])
 {
 	char *endptr;
-	unsigned int major, minor, argsc = 0;
-	cmdarg_t *args;
+	unsigned int major, minor;
 
-	argsc = cmd_getArgs(s, ". \t", &args);
-	if (argsc == 0) {
-		log_error("\nArguments have to be defined");
+	if (argc == 1) {
+		log_error("\n%s: Arguments have to be defined", argv[0]);
 		return -EINVAL;
 	}
-	else if (argsc != 2) {
-		log_error("\nWrong args: %s", s);
+	else if (argc != 2) {
+		log_error("\n%s: Wrong argument count", argv[0]);
 		return -EINVAL;
 	}
 
+
 	/* Get major/minor */
-	major = lib_strtoul(args[0], &endptr, 0);
-	if (*endptr) {
-		log_error("\nWrong major value: %s", args[0]);
+	major = lib_strtoul(argv[1], &endptr, 0);
+	if (*endptr != '.') {
```

Comment:
Why did it work before?

Reply:
Previously, there were `cmd_getArgs(s, ". \t", &args)` that tokenized the string of arguments, but the ` ` and `\t` separators have already been removed in `cmd_parse()`, so ` ` `\t` characters never occurred, but the only `.` character, so `cmd_getArgs` split the string with using `.` delimiter into two strings. It worked, but I think the proposed solution in this PR  is clearer.